api4/ldap_test.go

```diff
@@ -28,3 +28,40 @@ func TestLdapSync(t *testing.T) {
 	_, resp = th.Client.SyncLdap()
 	CheckForbiddenStatus(t, resp)
 }
+
+func TestGetLdapGroups(t *testing.T) {
```

Comment:
These tests don't look especially useful.

Reply:
Agreed, I was actually just going to add a note that I'm in the process of adding to them.